tools/generate_hotkeys.py

```diff
@@ -1,8 +1,17 @@
 #!/usr/bin/env python3
-from pathlib import Path, PurePath
+import pathlib
+import re
+import sys
+from importlib.machinery import SourceFileLoader
 
-from zulipterminal.config.keys import HELP_CATEGORIES, KEY_BINDINGS
 
+ROOT_DIRECTORY = pathlib.Path(__file__).resolve().parent.parent
+keys_file = SourceFileLoader(
+    "keys", f"{ROOT_DIRECTORY}/zulipterminal/config/keys.py"
+).load_module()
+HELP_CATEGORIES = keys_file.HELP_CATEGORIES
+KEY_BINDINGS = keys_file.KEY_BINDINGS
```

Comment:
What's the motivation for this refactoring?

Reply:
GitHub Actions cannot import from ``zulipterminal.config.keys`` thus the refactoring of imports.

Reply:
This kind of explanation belongs in the commit text, and possibly a comment in the file.

Unless I missed it, you didn't discuss what error you may have had and why you took this approach.

It's good you separated this out into another commit, since it's clearer it's a refactor.